### ISSUE TITLE
No-Op Conversions

### ISSUE THREAD

**Issue by cmoesel** [cqframework/clinical_quality_language#119]
Previous versions of the cql-to-elm translator allowed CQL such as this:
```
define quantityToQuantity: convert 10 'g' to Quantity
```

The newest version (1.2) now creates an error:
```
Could not resolve conversion from type System.Quantity to type System.Quantity.
```

Is this a regression or does cql-to-elm now intentionally disallow conversions that are essentially no-ops (when source and target have the same type)?

**Comment by cmoesel** [comment]
Looking at this further, some no-op conversions go through CQL-to-ELM just fine.  For example, the following compile with _no_ errors:
```
define stringToStr: convert 'str' to String
define intToInt: convert 10 to Integer
define booleanToBoolean: convert true to Boolean
define dateToDate: convert @2015-01-02 to DateTime
define timeToTime: convert @T11:57 to Time
```

So this does seem that it would be a bug in the case of Quantities.

**Comment by brynrhodes** [comment]
Addressed by pull request #123
